corehq/apps/geospatial/tests/test_views.py

```diff
@@ -28,6 +29,15 @@ def setUpClass(cls):
         )
         cls.webuser.save()
 
+        case_type = CaseType(domain=cls.domain, name='case_type')
+        case_type.save()
+        cls.case_prop_name = 'gps_prop'
+        CaseProperty(
+            case_type=case_type,
+            name=cls.case_prop_name,
+            data_type=CaseProperty.DataType.GPS,
+        ).save()
```

Comment:
Do we need to add these in tearDownClass or they are handled by domain delete?

Reply:
I have added deleting the case types, which should also take care of the associated case properties ([c4b7f00](https://github.com/dimagi/commcare-hq/pull/33426/commits/c4b7f00dfe9bd6b2394ec008fef64b332f2b534b)).